pyiron_base/jobs/datamining.py

```diff
@@ -34,11 +34,18 @@
 
 
 def _to_pickle(hdf, key, value):
-    hdf[key] = codecs.encode(pickle.dumps(value), "base64").decode()
+    hdf[key] = codecs.encode(
+        cloudpickle.dumps(obj=value, protocol=5, buffer_callback=None), "base64"
```

Comment:
@pmrv I fixed the protocol version to guarantee we always use the same version. Still the load() function seems to be already sufficiently flexible to choose the right version for loading automatically: https://docs.python.org/3/library/pickle.html#pickle.HIGHEST_PROTOCOL